src/layout/encode.ts

```diff
@@ -61,7 +61,7 @@ export default async function(type: Event): Promise<void> {
                                 tokens.push(value.id);
                                 if (value.parent && active) { tokens.push(value.parent); }
                                 if (value.next && active) { tokens.push(value.next); }
-                                metadata.push(data[key]);
+                                metadata.push(value.position ? `${data[key]}.${value.position}` : data[key]);
```

Comment:
>. [](start = 76, length = 1)

Seems too class to notation used for class name in CSS selectors. Could we use another notation or separator like @ to similar.

Reply:
Yes good idea - let me change it.

Reply:
I'll use ~ because we don't have to encode it in the URL; if we choose to later.